s/nub-keyboard/utils/keys.ts

```diff
@@ -0,0 +1,225 @@
+export interface KeyData {
+	key: string,
+	actionName: string | undefined,
+	data: string
+}
+
+export interface Key {
+	[key: string]: KeyData
+}
+
+export const keys: Key =
+{
+	"1": {
```

Comment:
maybe there are different ways we could organize our data about inputs, actions, keys, and bindings?